shell/components/nav/TopLevelMenu.vue

```diff
@@ -365,8 +364,12 @@ export default {
         content = this.shown ? null : contentText;
 
       // if key combo is pressed, then we update the tooltip as well
-      } else if (this.routeCombo && isObject(item) && item.ready) {
-        contentText = 'Switch clusters and keeps location';
+      } else if (this.routeCombo &&
+        typeof item === 'object' &&
+        !Array.isArray(item) &&
```

Comment:
why wouldn't array be a valid type here? it looks like lodash recognized them as objects https://lodash.com/docs/4.17.15#isObject

Reply:
It can be either a `string` or an `object` and logic changes based on that (different applications of the same function). 

Ah 😅 , I was missusing the `isObject` lodash function here! I though it ruled out array's... I understand why they allow it, but it it's kinda misleading in the sense that they are usually helper functions for stuff just like this 😛